CHANGELOG.md

```diff
@@ -5,6 +5,157 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.0.0
+
+_Apr 16, 2025_
+
+We're excited to [announce the first v8 stable release](https://mui.com/blog/mui-x-v8/)! 🎉🚀
+
+This is now the officially supported major version, where we'll keep rolling out new features, bug fixes, and improvements.
+Migration guides are available with a complete list of the breaking changes:
```

Comment:
Should we add the reference for migrating Material UI? and encourage people to migrate to Material UI v7

Reply:
Added 👍🏻